### ISSUE TITLE
Changelog document somehow converting `**` to `*-`

### ISSUE THREAD

**Issue by wilsonehusin** [kubernetes/release#2055]
#### What happened:

In the process of converting release notes from PR body to Markdown document, it seems that if there is an instance of `**`, they will be converted to `*-`.

#### What you expected to happen:

Things to remain.

#### How to reproduce it (as minimally and precisely as possible):

Generate release notes through krel.

#### Anything else we need to know?:

Most recently found in https://github.com/kubernetes/kubernetes/pull/101452, but it seems to have existed for a while

<details>

```
❯ ag '\*\w*-'
CHANGELOG-1.6.md
1832:* **- Add tcp/udp userspace proxy support for Windows. ([#41487](https://github.com/kubernetes/kubernetes/pull/41487), [@anhowe](https://github.com/anhowe))**

CHANGELOG-1.13.md
1703:* Fix missing flags in *-controller-manager --help. ([#71298](https://github.com/kubernetes/kubernetes/pull/71298), [@stewart-yu](https://github.com/stewart-yu))

CHANGELOG-1.21.md
1028:- Stdin is now only passed to client-go exec credential plugins when it is detected to be an interactive terminal. Previously, it was passed to client-go exec plugins when **stdout*- was detected to be an interactive terminal. ([#99654](https://github.com/kubernetes/kubernetes/pull/99654), [@ankeesler](https://github.com/ankeesler)) [SIG API Machinery and Auth]
1327:- New admission controller "denyserviceexternalips" is available.  Clusters which do not *need- the Service "externalIPs" feature should enable this controller and be more secure. ([#97395](https://github.com/kubernetes/kubernetes/pull/97395), [@thockin](https://github.com/thockin)) [SIG API Machinery]
1356:- Kubeadm: fix bad token placeholder text in "config print *-defaults --help" ([#98839](https://github.com/kubernetes/kubernetes/pull/98839), [@Mattias-](https://github.com/Mattias-)) [SIG Cluster Lifecycle]

CHANGELOG-1.18.md
2537:- `kubectl` no longer defaults to `http://localhost:8080`.  If you own one of these legacy clusters, you are *strongly- encouraged to secure your server.   If you cannot secure your server, you can set `KUBERNETES_MASTER` if you were relying on that behavior and you're a client-go user. Set `--server`, `--kubeconfig` or `KUBECONFIG` to make it work in `kubectl`. ([#86173](https://github.com/kubernetes/kubernetes/pull/86173), [@soltysh](https://github.com/soltysh)) [SIG API Machinery, CLI and Testing]

CHANGELOG-1.9.md
158:      - [**Kube-Proxy**](#kube-proxy)
1560:#### **Kube-Proxy**
1953:* Adds to **kubeadm upgrade apply**, a new **--etcd-upgrade** keyword. When this keyword is specified, etcd's static pod gets upgraded to the etcd version officially recommended for a target kubernetes release. ([#55010](https://github.com/kubernetes/kubernetes/pull/55010), [@sbezverk](https://github.com/sbezverk))

CHANGELOG-1.19.md
3702:- `kubectl` no longer defaults to `http://localhost:8080`.  If you own one of these legacy clusters, you are *strongly- encouraged to secure your server.   If you cannot secure your server, you can set `KUBERNETES_MASTER` if you were relying on that behavior and you're client-go user. Set `--server`, `--kubeconfig` or `KUBECONFIG` to make it work in `kubectl`. ([#86173](https://github.com/kubernetes/kubernetes/pull/86173), [@soltysh](https://github.com/soltysh)) [SIG API Machinery, CLI and Testing]
```

</details>

#### Environment:

- Cloud provider or hardware configuration: -
- OS (e.g: `cat /etc/os-release`): -
- Kernel (e.g. `uname -a`): -
- Others: -

**Comment by saschagrunert** [comment]
I think it's this one: https://github.com/kubernetes/release/blob/b6dd9aa61f79019382c2e97f3bbd84383dd1d25c/pkg/notes/notes.go#L808-L810

**Comment by saschagrunert** [comment]
Fix incoming in #2056
